src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container" style="width: 700px;margin-top: 5rem;">
-<div *ngFor="let data of sampleData">
+<div *ngFor="let data of sampleData | async">
```

Comment:
use descriptive name, what are these data?

Reply:
Please mark observables with $ at the end, e.g. sampleData$

Reply:
Updated